From aca26ef8a32de55fb434141e563035cbf31bbb72 Mon Sep 17 00:00:00 2001
From: Yosef Or Boczko <yoseforb@gmail.com>
Date: Tue, 17 Dec 2013 21:29:03 +0200
Subject: Fix the size of the default text and of the search entry

Also fix the size of the label in the run-dialog.

This for infinality.

Signed-off-by: Yosef Or Boczko <yoseforb@gmail.com>
---
 data/theme/gnome-shell.css | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/theme/gnome-shell.css b/data/theme/gnome-shell.css
index 1ba8250..2547d5d 100644
--- a/data/theme/gnome-shell.css
+++ b/data/theme/gnome-shell.css
@@ -22,7 +22,7 @@
 /* default text style */
 stage {
     font-family: cantarell, sans-serif;
-    font-size: 11pt;
+    font-size: 10pt;
     color: white;
 }
 
@@ -416,7 +416,7 @@ StScrollBar StButton#vhandle:active {
 .modal-dialog StEntry {
     color: rgb(64, 64, 64);
     caret-color: rgb(64, 64, 64);
-    font-size: 12pt;
+    font-size: 11pt;
     caret-size: 1px;
     selected-color: white;
     padding: 4px 12px;
@@ -1895,7 +1895,7 @@ StScrollBar StButton#vhandle:active {
 /* Run Dialog */
 
 .run-dialog-label {
-    font-size: 12pt;
+    font-size: 11pt;
     font-weight: bold;
     color: #999999;
     padding-bottom: .4em;
-- 
1.8.5.1

